mods/gen4/formats-data.js

```diff
@@ -1942,19 +1942,19 @@ let BattleFormatsData = {
 		tier: "OU",
 	},
 	rotomwash: {
-		randomBattleMoves: ["thunderbolt", "substitute", "painsplit", "shadowball", "hiddenpowerice", "willowisp", "trick", "hydropump"],
+		randomBattleMoves: ["thunderbolt", "substitute", "painsplit", "shadowball", "hiddenpowerice", "willowisp", "trick", "hydropump", "rest", "sleeptalk"],
```

Comment:
idk if that's a good idea. There's like 3 moves that don't suck with RestTalk, WoW/TBolt/SBall.

Reply:
I generated a lot of sets and didn't see any bad RestTalk ones. It always gets Thunderbolt due to the check on line 380, and any one of WoW/SBall/HPIce/Hydro seem fine alongside that. That even mirrors the RestTalk set at https://www.smogon.com/dex/dp/pokemon/rotom-wash/